lnd.go

```diff
@@ -77,6 +81,24 @@ func lndMain() error {
 	}
 	defer chanDB.Close()
 
+	// Create the macaroon authentication/authorization service.
+	macaroonService, err := macaroons.NewService(cfg.DataDir)
+	if err != nil {
+		srvrLog.Errorf("unable to create macaroon service: %v", err)
+		return err
+	}
+
+	// Create macaroon files for lncli to use if they don't exist.
+	if !fileExists(cfg.AdmMacPath) && !fileExists(cfg.ReadMacPath) {
```

Comment:
It assumes that this is intentional (specifically, you can remove `admin.macaroon` and only have a read-only macaroon available, which will be really nice for autopilot to collect fees).